recipes-bsp/grub/grub-efi_git.bbappend

```diff
@@ -2,7 +2,7 @@ require grub-nilrt.inc
 
 GRUB_BUILDIN += "smbios chain multiboot efi_uga font gfxterm gfxmenu terminal \
                 minicmd iorw echo reboot terminfo loopback memdisk tar help serial \
-                ls search_fs_uuid udf btrfs ntfs reiserfs xfs lvm ata tpm \
+                ls search_fs_uuid udf btrfs ntfs reiserfs xfs lvm ata \
```

Comment:
Is this specific to the measured boot feature? I imagine that we might want this `tpm` option in general.

Reply:
It was added in the same time period by Haris in Penguin CL:6218826 when we added the measured boot feature, but there was no explanation in the CL.